ajax/add-ftp-user.php

```diff
@@ -0,0 +1,37 @@
+<?php
+require('../template/top.php');
+require('../api/groupme-funcs.php');
```

Comment:
I don't think we're calling groupme anywhere in this script

Reply:
taken out